services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeBuilder.kt

```diff
@@ -136,9 +135,7 @@ data class HankeBuilder(
                 Names(
                     firstName = perustaja.etunimi,
                     lastName = perustaja.sukunimi,
-                    givenName = perustaja.etunimi
-                )
-        )
+                    givenName = perustaja.etunimi))
```

Comment:
I've been fixing these by adding the missing trailing comma.

Reply:
I added some commas, not quite sure where they should be added and where not.

Reply:
If it's a list that spans many rows, a trailing comma should be added. Beyond that, it depends on the situation.

Reply:
ktfmt added automatic trailing commas, but only for the `google` style. I'm almost tempted to adapt it, but that would change nearly every line of code since it's so different, starting from having 2 space indentations instead of 4.